acceptance/testdata/analyzer/app-image/Dockerfile

```diff
@@ -1,6 +1,4 @@
-ARG fromImage
-
-FROM $fromImage
+FROM ubuntu:bionic
```

Comment:
Not necessary for this PR but for some (not all) of these test fixtures I wonder of building them directly with `imgutil` would be easier than managing `Dockerfile` test fixture (and eventually parameterizing for windows etc).